compose.yml

```diff
@@ -90,25 +90,129 @@ services:
+    restart: unless-stopped
+    command: server
+    environment:
+      AUTHENTIK_SECRET_KEY: ${AUTHENTIK_SECRET_KEY:?secret key required}
+      AUTHENTIK_REDIS__HOST: redis_authentik
+      AUTHENTIK_POSTGRESQL__HOST: postgres_authentik
+      AUTHENTIK_POSTGRESQL__USER: authentik
+      AUTHENTIK_POSTGRESQL__NAME: authentik_db
+      AUTHENTIK_POSTGRESQL__PASSWORD: ${PG_PASS}
+    volumes:
+      - ./authentik/media:/media
+      - ./authentik/custom-templates:/templates
+    env_file:
+      - .secrets/authentik_admin_password.env
+    user: "${USER_ID}:${GROUP_ID}"
+    networks:
+      - authentik
       - caddy_proxy
     labels:
-      caddy_0: s3.fabricant.ucsd.edu
-      caddy_0.reverse_proxy: "{{upstreams 9000}}"
-      caddy_1: mc.fabricant.ucsd.edu
-      caddy_1.reverse_proxy: "{{upstreams 9001}}"
+      caddy: auth.fabricant.ucsd.edu
+      caddy.reverse_proxy: "{{upstreams 9000}}"
+    depends_on:
+      postgres_authentik:
+        condition: service_healthy
+      redis_authentik:
+        condition: service_healthy
+  authentik_worker:
+    image: ghcr.io/goauthentik/server:2025.6.4
+    restart: unless-stopped
+    command: worker
+    environment:
+      AUTHENTIK_SECRET_KEY: ${AUTHENTIK_SECRET_KEY:?secret key required}
+      AUTHENTIK_REDIS__HOST: redis_authentik
+      AUTHENTIK_POSTGRESQL__HOST: postgres_authentik
+      AUTHENTIK_POSTGRESQL__USER: authentik
+      AUTHENTIK_POSTGRESQL__NAME: authentik_db
+      AUTHENTIK_POSTGRESQL__PASSWORD: ${PG_PASS}
+    # `user: root` and the docker socket volume are optional.
+    # See more for the docker socket integration here:
+    # https://goauthentik.io/docs/outposts/integrations/docker
+    # Removing `user: root` also prevents the worker from fixing the permissions
+    # on the mounted folders, so when removing this make sure the folders have the correct UID/GID
+    # (1000:1000 by default)
+    user: root
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ./authentik/media:/media
+      - ./authentik/certs:/certs
+      - ./authentik/custom-templates:/templates
+    env_file:
+      - .secrets/authentik_admin_password.env
+    networks:
+      - authentik
+    depends_on:
+      postgres_authentik:
+        condition: service_healthy
+      redis_authentik:
+        condition: service_healthy
+  
+
+  # minio:
+  #   image: quay.io/minio/minio
+  #   container_name: minio
+  #   restart: always
+  #   volumes:
+  #     - ./minio:/data
+  #   environment:
+  #     MINIO_ROOT_USER: e4e_minio_admin
+  #     MINIO_ROOT_PASSWORD_FILE: /run/secrets/minio_admin_password
+  #   command: server /data --console-address ":9001"
+  #   secrets:
+  #     - minio_admin_password
+  #   networks:
+  #     - caddy_proxy
+  #   labels:
+  #     caddy_0: s3.fabricant.ucsd.edu
+  #     caddy_0.reverse_proxy: "{{upstreams 9000}}"
+  #     caddy_1: mc.fabricant.ucsd.edu
+  #     caddy_1.reverse_proxy: "{{upstreams 9001}}"
```

Comment:
Any reason the minio stuff is commented rather than deleted, they aren't going in a direction that makes me feel confident we would return to it

Reply:
I want it for reference when we replace it.  I have no intention of returning to minio.